polygeist/lib/Dialect/Polygeist/Transforms/LoopInternalization.cpp

```diff
@@ -39,9 +39,13 @@ namespace polygeist {
 using namespace mlir;
 using namespace mlir::polygeist;
 
-static llvm::cl::list<unsigned> LoopInternalizationTileSizes(
-    DEBUG_TYPE "-tile-sizes", llvm::cl::CommaSeparated,
-    llvm::cl::desc("Tile sizes used in LoopInternalization"));
+static llvm::cl::list<unsigned>
```

Comment:
What's the reason to declare these operations here and not as options of the loop internalization pass?

Reply:
Good question, when I work in LLVM opt, we always do it this way. Do you know if there is any advantages to do them as pass options?

Reply:
I think it's the canonical way of adding pass-specific options in MLIR and it also gives a nice scoping (i.e., reduces the danger of name clashes), as the options are specified like this `--detect-reduction='use-opaque-pointers=1'`. 

The loop internalization pass even already has such an option for relaxed aliasing, so I think we should stick to that mechanism for the other options.

Reply:
done.